CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.0.12
+
+### New features
+* `ResultSet.query_id` contains a unique query identifier that might be useful for retrieving query metrics from `system.query_log`
+
+### Breaking changes
+* `client.insert` now returns `string` (query_id) instead of `void`
```

Comment:
Why not `{ query_id: string }` to make it extendable in the future?

Reply:
that's great